apps/src/componentLibrary/common/contexts/DropdownContext.tsx

```diff
@@ -0,0 +1,24 @@
+import React, {createContext, useContext, useState, ReactNode} from 'react';
+
+const DropdownContext = createContext({
+  activeDropdownName: '',
+  setActiveDropdownName: (name: string) => {
+    console.log(name);
```

Comment:
Intentional logging?

Reply:
That's just a placeholder to mach typescript types, not used anywhere. Replace with     `return name as unknown as void;` , also added explaining jsdoc comment above the function